README.md

```
@@ -363,9 +376,33 @@ pub struct Declaration  L10-120
+symbol you can name from a tree you haven't mapped yet:
+
+```bash
+ast-bro show src/ TakeDamage            # searches every parseable file
+ast-bro show 'src/**/*.cs' TakeDamage   # quote it — see below
+ast-bro show a.java b.java greet        # an explicit list works too
+```
+
+A multi-file answer leads with its coverage, and caps rendered bodies at
+`--limit 20` while still reporting the true total:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)